src/main/java/hudson/plugins/clover/Ratio.java

```diff
@@ -62,7 +64,16 @@ public String getPcWidth() {
 
     public String getPcUncovered() {
         float pcUncovered = 100.0f - getPercentageFloat();
-        return pcFormat(pcUncovered);
+
+        /* Since this is the negative of getPcCovered() we need to invert the
+         * rounding mode to ensure it most closely complements getPcCovered() */
+        PC_WIDTH_FORMAT.setRoundingMode(RoundingMode.UP);
```

Comment:
this is not correct. you shall not modify static variables from non-static context. we can have flaky behaviour when running this code from multiple threads. 

instead of this, I suggest creating two static variables, e.g.:
 * rename PC_WIDTH_FORMAT to PC_ROUND_DOWN_FORMAT
 * add PC_ROUND_UP_FORMAT

and use the latter one for getPcUncovered()